dev_mode/package.json

```diff
@@ -4,8 +4,11 @@
   "private": true,
   "scripts": {
     "build": "webpack",
+    "build:dev": "jlpm run build",
+    "build:dev:minimize": "jlpm run build:dev",
```

Comment:
Will this actually do a dev build with minimization turned on?

Reply:
No, the `minimize` there is like a no-op: `jlpm build:dev:minimize` will just call `jlpm build:dev`. I included it 

- A. to help simplify the semantics of handling multiple options, especially if we ever add another one beyond `build-dev` and `minimize`
- B. as a placeholder, in case someone comes up with a good reason why a minified dev build should exist